airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/TimeStringToIntegral.kt

```
@@ -0,0 +1,103 @@
+        return zdt.toInstant().toEpochMilli() * 1_000
+    }
+
+    private fun toEpochMicrosWithoutTimezone(timestampString: String): Long {
+        val dt = LocalDateTime.parse(timestampString, DATE_TIME_FORMATTER)
+        return dt.toInstant(ZoneOffset.UTC).toEpochMilli() * 1_000
+    }
+
+    private fun toEpochMicros(timestampString: String): Long {
+        return try {
+            toEpochMicrosWithTimezone(timestampString)
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)